src/docs/rialto-servermanager-api.json

```diff
@@ -0,0 +1,314 @@
+        "StateChangeEvent"
+      ],
+      "example": "StateChangeEvent"
+    },
+    "eventId": {
+      "description": "Id of the listener",
+      "type": "string",
+      "example": "events.1"
+    },
+    "listener": {
+      "description": "Listener Info",
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        }
+      },
+      "required": [
+        "eventType"
+      ]
+    },
+    "activateRequest": {
+        "type": "object",
+        "default": null,
+        "description": "",
+        "properties": {
+          "id": {
+            "$ref": "#/definitions/id"
+          }
+        },
+        "required": [
+          "id"
+        ]
+    },
+    "stopRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "deactivateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "getStateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "registerRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "event",
+        "id"
+      ]
+    },
+    "unregisterRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "activateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "socket": {
+          "$ref": "#/definitions/socket"
+        }
+      },
+      "required": [
+        "socket"
+      ]
+    },
+    "stateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/state"
+        }
+      },
+      "required": [
+        "state"
+      ]
+    },
+    "getListenersResult": {
+      "type": "object",
+      "properties": {
+        "listeners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/listener"
+          }
+        }
+      },
+      "required": [
+        "listeners"
+      ]
+    }
+  },
+  "methods": {
+    "activate": {
+      "summary": "Activate an app session and spawn a socket",
```

Comment:
"summary": "Activates an app session.\n\nSpawns a server instance if app session was in 'notrunning' state.\nIf App session server state changes, then StateChangeEvent 'active' will be sent.\nIf App session server is already in 'active' state, then method call will succeed but no StateChangeEvent will be sent.",